app/controllers/stories/feeds_controller.rb

```diff
@@ -43,7 +43,7 @@ def signed_in_base_feed
                  controller: self,
                  page: @page,
                  tag: params[:tag],
-                 number_of_articles: 25,
+                 number_of_articles: 45,
```

Comment:
This isn't a new experiment, but is an adjustment to the number of articles returned — since the posts that return via pagination are not very relevant due to reasons, for now it makes sense to show more here for more total relevant posts returned.